admin/css/admin.css

```diff
@@ -427,6 +427,14 @@ body.post-type-volunteer_opp .wp-pointer-arrow {
 }
 
 /**
+* Widget styling
+*/
+.wi-widget-block-label:after {
```

Comment:
@jg314 

The browser support is as follows for the after pseudo element: 
Chrome: Supported
IE: 8+
Firefox: 1.0+
Opera: 4+
Safari: 4.0+

Do we need to support any further back than IE 8?